dashboard/test/controllers/api/v1/pd/workshop_enrollments_controller_test.rb

```diff
@@ -270,7 +270,7 @@ class Api::V1::Pd::WorkshopEnrollmentsControllerTest < ::ActionController::TestC
       role: "Counselor",
       grades_teaching: ["Kindergarten", "Grade 1", "Grade 2"],
       attended_csf_intro_workshop: "No",
-      csf_course_experience: {"Course A": "none", "Course B": "a few lessons", "Course E": "most lessons"},
+      csf_course_experience: {'Course A': "none", 'Course B': "a few lessons", 'Course E': "most lessons"},
       csf_courses_planned: ["Course E", "Course F"],
```

Comment:
So why does this apply to `"Course A"` but not `"Course E"`?

Reply:
I assume you're asking about `csf_courses_planned` on the next line? This particular rule only checks Symbols, and those are Strings. It's definitely not obvious, but with this particular Ruby syntax, `Course A` is a Symbol, not a String; you have to do something like `{"Course A" => "none"}` if you want String keys in your hashes.